jwst/pipeline/calwebb_ami3.py

```diff
@@ -89,16 +89,13 @@ def process(self, input):
             # Save the LG analysis results to a file
             result.meta.asn.pool_name = asn['asn_pool']
             result.meta.asn.table_name = op.basename(asn.filename)
-            output_file = self.save_model(
-                result, output_file=input_file, suffix='ami', acid=acid,
-            )
-            self.log.info('LG results saved to %s', output_file)
+            self.save_model(result, output_file=input_file, suffix='ami')
```

Comment:
This needs to be restored to the original code. These are "Level2c" names which required the ACID in them.